### PR TITLE
Fix: 1719 Columns are misaligned on Payments->Transactions/Disputes page

### LINKED ISSUE #1719: [GlobalStep] Columns are misaligned on Payments->Transactions/Disputes page
## Bug Description:
"Fees/Net/Order#/Source/Deposit" columns are misaligned on "Payments->Transactions" page
"Status/Reason/Order #/Source/Respond by "columns are misaligned on "Payments->Disputes" page

## Environment:
Woocommerce Version : WooCommerce 5.2.2 
WooCommerce Payments: v2.4.0
WooCommerce Subscriptions: v3.0.12
WordPress version: (v5.7)

**PC**:
Windows 10, Mac 10.14.6
Chrome(Version 90.0.4430.93) 
Firefox(Version 87.0)
Safari: v13.1.1

## Steps To Reproduce:
1. Create any test site using JN site.
1. Install and activate all the required plugins 
1. Complete the onboarding setup wizard setup 
1. Go to Plugins->Add New and upload the zip files for WooCommerce Payments v2.4.0 plugin
1. Activate WooCommerce Payments v2.4.0 
1. Complete the wooCommerce Payments setup
1. Complete "Get ready to start selling" section on WooCommerce->Home Page
1. Purchase products from front-end
1. Go to WooCommerce->Orders page and check all the orders are with "Processing" status
1. Go to the "Payments->Transactions" page
1. Note that "Fees/Net/Order#/Source/Deposit'" columns are misaligned on the "Payments->Transactions" page
1. Go to the "Payments->Disputes" page
1. Observe that "Status/Reason/Order #/Source/Respond by "columns are misaligned on "Payments->Disputes" page

## Actual Result:
 Columns are misaligned on Payments->Transactions/Disputes page

## Expected Result:
Columns should be properly aligned on the Payments->Transactions/Disputes page

## ScreenShot:
#1719_Disputes
#1719_Transactions

**Isolating the problem (mark completed items with an [x]):**
- [ ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [ ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

### PR DESCRIPTION
**Pull request by naman03malhotra**
Fixes #1719 

#### Changes proposed in this Pull Request
As the issue reported indicates that the table header items are right-aligned which was due to the default alignment of the table component. To fix this, I overrode the default CSS with a scoped CSS for transactions and dispute tables to make them center-aligned. A scoped alignment won't affect the alignment of the component used at other places.
- override alignment for transactions list page
- override alignment for disputes list page
- slight adjustment for table header items with arrow icon

#### After fix
image

image

#### Testing instructions
- checkout to `fix/1719-columns-misaligned`
- visit transactions and dispute pages
- Items should be aligned now.
-------------------

- [x] Added changelog entry (or does not apply)
- [ ] Covered with tests (or have a good reason not to test in the description ☝️)
- [ ] Tested on mobile (or does not apply)

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -7,6 +7,7 @@
 * Fix - WooCommerce Payments disappeared from WooCommerce Settings if WooCommerce Subscriptions is activated.
 * Add - REST endpoint to capture payments by order ID
 * Add - Payment request button support for checkout and cart blocks.
+* Fix - Columns are misaligned on Payments->Transactions/Disputes page #1719.
```
Comment (naman03malhotra) [inline changelog.txt:10]: Discussion to include the issue number in changelog is happening here: https://github.com/Automattic/woocommerce-payments/pull/1741#discussion_r627920439

File: client/disputes/style.scss [hunk 2]
```diff
@@ -1,3 +1,11 @@
+// dispute list table css override
+.disputes-list {
```
Comment (shendy-a8c) [inline client/disputes/style.scss:2]: I wonder if this CSS class is better prefixed with `wcpay-` like the other classes below it.

**Review by shendy-a8c — CHANGES_REQUESTED** [review]
I tested and worked as expected. I left some feedback.

**Review by shendy-a8c — APPROVED** [review]
LGTM.

**Pull request merged**